src/applications/accredited-representative-portal/components/common/Header/common/OfficialGovtWebsite.jsx

```diff
@@ -44,7 +44,7 @@ const OfficialGovtWebsite = ({ isMobile }) => {
                   data-testid="official-govt-website-toggle-wider-than-mobile"
                   className="usa-accordion-button usa-banner-button"
                   aria-expanded={isExpanded}
-                  aria-controls="gov-banner"
+                  aria-controls="gov-banner gov-banner-mobile"
```

Comment:
i don't think this should be here based on the ternary. only one is going to be mobile

Reply:
Both aria-controls targets will be on the page but display none or block based on media queries. Use case is toggling the expanded-ness interleaved with shrinking and growing viewport.

Reply:
ah, that makes sense. was skimming through too quickly. thanks!

Reply:
Hmm looks like that's not even the behavior I'm achieving. Let me play with the logic. But what do you think is the desired behavior of interleaving expand/collapse w/ shrink/grow? One of the more immediate options is for expanded-ness to be maintained across growing/shrinking viewport if I can pull that off easily in code.

Reply:
The interleaving expanded state between shrink/grow feels like an edge case to me personally but I would love to simplify these mobile/not-mobile component splits in general.  I know in Will's PR for https://github.com/department-of-veterans-affairs/vets-website/pull/31615 some of that is addressed already for the footer. Maybe not worth changing too much in this stack of PR to minimize conflicts later

Reply:
Agreed. BTW I didn't see the mobile, nonmobile elements toggling in concert because they are two components with two state hooks. I'd resolve that by lifting state to parent, so I'll just do that in a separate commit as an indication that we still have the remote possibility of caring about this even though we will likely determine that is way too minuscule to care about.